db/migrations/20230202110100-add-repo-sync-state-fk.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+module.exports = {
+	up: async (queryInterface) => {
+		await queryInterface.addConstraint("RepoSyncStates", ["subscriptionId"], {
+			type: "foreign key",
+			name: "fk_reposyncst_subs_subid",
```

Comment:
This name isn't very descriptive.  Do you really need to specify the owning table?  Why not just `fk_subscriptionId_subscriptions`?

Reply:
It is bit confusing over the web, so I found this [one](https://stackoverflow.com/a/65206388), it seems if the `constraint` is backed by an index, which is the case for unique constraint, its naming should be global across the schema, hence I added the table name.

Reply:
ah right.  Is there a character limit?  I'd rather make it longer and precise than abbreviated and can be confused